frontend/src/app/modules/perflow-studio/playlists-page/playlists-page.component.ts

```diff
@@ -1,10 +1,35 @@
 import { Component } from '@angular/core';
+import { filter } from 'rxjs/operators';
+import { PlaylistView } from 'src/app/models/playlist/playlist-view';
+import { AuthService } from 'src/app/services/auth/auth.service';
+import { PlaylistsService } from 'src/app/services/playlists/playlist.service';
 
 @Component({
   selector: 'app-playlists-page',
   templateUrl: './playlists-page.component.html',
   styleUrls: ['./playlists-page.component.sass']
 })
 export class PlaylistsPageComponent {
+  playlists: PlaylistView[] = [];
+  userId: number = 1;
```

Comment:
Maybe set to -1 to get an exception if something goes wrong instead of displaying data of the first user?